### PR TITLE
Fix DrawerLayoutAndroid not able to set opacity

### PR DESCRIPTION
**Pull request by holyxiaoxin**
I'm just cleaning up my branch, but please refer to https://github.com/facebook/react-native/pull/6948.

### CODE REVIEW

**Comment by holyxiaoxin** [comment]
@christopherdro @janicduplessis I'm just cleaning up my branch, didn't did it the right way the first time.
So, we're just waiting for the master branch test to pass right? Thanks.

**Comment by christopherdro** [comment]
@holyxiaoxin Your commits will all be squashed once the PR gets shipped. No need to keep opening up new PR's.

**Comment by holyxiaoxin** [comment]
Oops, was wondering why I did `rebase -i` and so many commits. My bad. Thanks!

**Comment by christopherdro** [comment]
:) No worries. You gotta make sure to force push to your branch after rebasing.

**Comment by janicduplessis** [comment]
Weird, the circleci build doesn't seem to get triggered by this PR.

**Comment by holyxiaoxin** [comment]
Does circleci only trigger on master branch?

File: Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js [hunk 1]
```diff
@@ -160,7 +178,12 @@ var DrawerLayoutAndroid = React.createClass({
   render: function() {
     var drawStatusBar = Platform.Version >= 21 && this.props.statusBarBackgroundColor;
     var drawerViewWrapper =
-      <View style={[styles.drawerSubview, {width: this.props.drawerWidth}]} collapsable={false}>
+      <View
+        style={[
+          styles.drawerSubview,
+          {width: this.props.drawerWidth, backgroundColor: this.props.drawerBackgroundColor }
```
Comment (janicduplessis) [inline Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js]: Small nit, can you just remove the extra space before the last bracket.

**Comment by janicduplessis** [comment]
Ok, can you just fix the small nit I pointed out then we can shipit, there are no test that test what you added anyway and the ones on travis ran fine so it's good.

**Comment by holyxiaoxin** [comment]
Done (:

**Comment by janicduplessis** [comment]
Thanks for the PR @holyxiaoxin! The tests actually ran on circle this time I'll just wait for that to complete then I'll ship it.

**Comment by holyxiaoxin** [comment]
roger

**Comment by holyxiaoxin** [comment]
Any ideas what's the warning message about below? It was passing a moment ago.

```
Link EndToEndTestTests (46 ms)
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
ld: warning: directory not found for option '-F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk/Developer/Library/Frameworks'
```

**Comment by janicduplessis** [comment]
Probably just a flacky test, only the objc test failed so it doesn't matter.

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**